src/coprocessor/dag/dag.rs

```diff
@@ -117,6 +154,18 @@ impl DAGContext {
         self.exec.collect_statistics_into(statistics);
     }
 
+    pub fn can_cache(&mut self) -> bool {
+        self.enable_distsql_cache && (self.has_aggr || self.has_topn)
```

Comment:
Why must need `has_aggr || has_topn`?

Reply:
In dist-sql cache design, the cache's purpose is to cache high CPU usage result. So I think we should cache aggr and topn dist-sql. For simple query, RocksDB already has cache for it.